.gitignore

```diff
@@ -5,3 +5,5 @@ myimage.yaml
 /.venv
 /ansible.cfg
 *.pyc
+.version
```

Comment:
I need to create a .version file to be able to do a local build. If I check it in by accident it will fluff up the entire build - as the presence of that file overrides all the rest of the versioning stuff we have in place. I figured the lesser evil was to add it to the .ignore file.